### PR TITLE
Show development as separate category in google analytics.

### PR DESCRIPTION
**Pull request by jonasem**
image

### CODE REVIEW

**Comment by jonasem** [comment]
Also compact log file data of usage data
image

File: src/utils/usageData.ts [hunk 1]
```diff
@@ -17,6 +17,9 @@ import {
 } from './persistentStore';
 
 const trackId = 'UA-22498474-5';
+const isDevelopment = process.env.NODE_ENV === 'development';
```
Comment (nicolai-nordic) [inline src/utils/usageData.ts:20]: This is the third way (at least) of checking if we are in development mode, I think we should agree on which we use :) I don't know which one is better, but we should definitely agree on what we want to use in the future.

Currently we are also using the `electron-is-dev` module in PPK and using `app.isPackaged` in `deviceLister.ts` also in shared.

**Review by nicolai-nordic — APPROVED** [review]
Other than landing on how we should check against dev environment it looks good to me!

File: src/utils/usageData.ts (hunk 1 above)
Comment (jonasem) [reply src/utils/usageData.ts:20]: Sounds like a job for shared!

Comment (jonasem) [reply src/utils/usageData.ts]: Decided to go with app.isPackaged.

**Pull request merged**